int64 real_X
int64 real_Y
int64 real_Z
int16 label